[OSDOCS#8997]: z-stream RN's for 4.11.55 (#69228)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

==== Updating

To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

[id="ocp-4-11-55"]
=== RHSA-2023:7691 {product-title} 4.11.55 bug fix update and security update

Issued: 2023-12-13

{product-title} release 4.11.55, which includes security updates, is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:7691[RHSA-2023:7691] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7693[RHBA-2023:7693] advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.11.55 --pullspecs
----

[id="ocp-4-11-55-bug-fixes"]
==== Bug fixes

> * Previously, upgrading to 4.12 on the OpenStack platform could fail if the master nodes were attached to additional networks. During the upgrade, there is a short moment when both providers are active at the same time and can report different node IPs. This is due to a known race condition when switching from an in-tree cloud provider to an external cloud provider. With this release, an annotation has been added that causes both providers to report the same primary node IP, which prevents node IP flapping. (link:https://issues.redhat.com/browse/OCPBUGS-20122[*OCPBUGS-20122*])

Reviewer: ```suggestion
* Previously, upgrading to 4.12 on the OpenStack platform could fail if the master nodes were attached to additional networks. During the upgrade, both providers are active at the same time for a short period and can report different node IPs. This behavior is due to a known race condition when switching from an in-tree cloud provider to an external cloud provider. With this release, an annotation is added that causes both providers to report the same primary node IP, which prevents node IP flapping. (link:https://issues.redhat.com/browse/OCPBUGS-20122[*OCPBUGS-20122*])
```
- Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences): "avoid using expletive constructions such as _it is_, _there are_, and _there is_, which hide the subject of the sentence."
- To avoid an ambiguous pronoun reference try to always include a verb after "This" or "That" to clarify what those pronouns refer to. See https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references.

Author: thank you!

[id="ocp-4-11-55-upgrading"]
==== Updating

> To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

Reviewer: ```suggestion
To update an {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
You can almost always remove "existing" and retain the same meaning.
